lab/graph_builder.py

```diff
@@ -32,9 +33,9 @@ def build_line_graph(self):
         self.plt.legend()
     
     # Function to add bar label/val to list of bars 
-    def add_bar(self, label, val):
-        self.bars.append((label, val))
+    def add_bar(self, val):
+        self.bar_vals.append(val)
     
     def build_bar_graph(self):
-        # TODO
-        pass
\ No newline at end of file
+        self.plt.bar(self.xlabels, [val for val in self.bar_vals])
```

Comment:
Let's just make this `self.plt.bar(self.xlabels, self.bar_vals)`, no need to make it an iterative process.

Reply:
That was embarrassing...